app/static/src/scss/includes/_search-transferring-body.scss

```diff
@@ -191,7 +191,7 @@ a.govuk-link {
 .govuk-button {
   &__search-filters-form-apply-button {
     height: 35px;
-    width: 70px;
+    width: 110px;
```

Comment:
why did this need changing?

Reply:
earlier the button text was 'Apply', now it has changed to 'Apply terms' as per the discussion with Terry on content changes, so has to accommodate the full text as per new changes

Reply:
cool 👍